Detections/MultipleDataSources/UserImpersonateByAAID.yaml

```diff
@@ -0,0 +1,67 @@
+id: 11c3d541-5fa5-49df-8218-d1c98584473b
+name: User impersonation by Identity Protection alerts
+description: |
+  'This detection focuses on identifying user-related events involving IAM roles, groups, user access, and password changes. It examines instances where the user's IP address matches and alerts generated by Identity Protection share the same IP address. The analysis occurs within a time window of 1 hour, helping to flag potential cases of user impersonation.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+  - connectorId: AzureActiveDirectoryIdentityProtection
+    dataTypes:
+      - SecurityAlert (IPC)
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1134

```

Comment:
I don't see where Token Manipulation is being looked for here

Reply:
"CreateAccessKey", "DeleteAccessKey"can be used for token or key manipulation